target/linux/ramips/image/Makefile

```diff
@@ -130,6 +130,11 @@ define Build/zyimage
 	$(STAGING_DIR_HOST)/bin/zyimage $(1) $@
 endef
 
+define Build/fonfxcimage
+	$(STAGING_DIR_HOST)/bin/uimage_padhdr -i $@ -o $@.new
```

Comment:
"$(STAGING_DIR_HOST)/bin/" might not be necessary anymore.

Reply:
changed at def4fb1